src/Microsoft.Maintainability.Analyzers/UnitTests/AvoidUnusedPrivateFieldsTests.Fixer.cs

```diff
@@ -12,22 +14,92 @@ public class AvoidUnusedPrivateFieldsFixerTests : CodeFixTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
+        }
+
+        [Fact]
+        public void CA1823CSharp()
+        {
+            VerifyCSharpFix(
+                @"  
+class C  
+{  
```

Comment:
What we should test "private int used1, unused1, used2;" gets converted to "private int used1, used2;" - that's a non-fix all test.